feature-libs/cart/base/components/added-to-cart-toast/added-to-cart-toast.component.html

```diff
@@ -0,0 +1,29 @@
+<ng-container *ngIf="cartToasts$ | async as cartToasts">
```

Comment:
How is accessibility handled?

The recommendation from the a11y guys was to use the "assistive message directive".